crates/ruff/resources/test/fixtures/flake8_bugbear/B029.py

```diff
@@ -0,0 +1,8 @@
+"""
+Should emit:
+B029 - on lines 7
+"""
+try:
+    x = 1
+except():
+    print("error")
```

Comment:
Can we replace this with the version from [flake8-bugbear](https://github.com/PyCQA/flake8-bugbear/blob/main/tests/b029.py) itself?

```py
"""
Should emit:
B029 - on lines 8 and 13
"""

try:
    pass
except ():
    pass

try:
    pass
except () as e:
    pass
```

Reply:
@carlosmiei sure!

Reply:
@charliermarsh sure!